persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MolecularDataMapper.xml

```diff
@@ -91,6 +91,19 @@
         </where>
     </select>
 
+    <!-- This routine is an abbreviated copy of getGeneMolecularAlterationsIter above. The two should be kept in sync. -->
```

Comment:
So what's the difference here? No projection checks? How much does this actually speed things up?

Reply:
No projection check or gene id nullity/empty check. Although it seems trivial, I was actually getting non-negligible speedups (not huge, but better than nothing). I think the reason is just because the table is quite large, so skipping those checks for every single row makes a difference